build.sh

```diff
@@ -180,8 +178,8 @@ release() {
     git checkout -f $1
     git submodule update --init --checkout
     tag=$(git describe --tags --abbrev=0 ${1})
```

Comment:
@lwcislo let's use the variable than make git search through history again and again. So:
```
    check_version $tag
    check_if_legacy $tag
```